### PR TITLE
Toast

### PR DESCRIPTION
**Pull request by kotalwar-krishna**
Toast component added.

Releated css you find tin toast.css.

screenshots:
toast

### CODE REVIEW

File: components/component.css [hunk 1]
```diff
@@ -30,7 +31,7 @@
     --light-grey-color:#666;
     --complementory-color:#CAF36C;
     --grey-dark: gray;
```
Comment (Nagashree01) [inline components/component.css:33]: you can use hex code here as other colors are also in hex-code

File: components/toast.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
+.toast-success-msg {
+    padding: 0.8rem;
```
Comment (psychoactiv) [inline components/toast.css:2]: suggestion: you can use px for values less than 1 rem

**Review by psychoactiv — APPROVED** [review]
Code looks clean

**Review by Nagashree01 — APPROVED** [review]
Suggested a change. Other than that everything is good.

File: components/component.css (hunk 1 above)
Comment (kotalwar-krishna) [reply components/component.css:33]: OK will do

**Review by sakleprajwal — APPROVED** [review]
Everything looks good. Please go ahead

File: components/toast.css (hunk 2 above)
Comment (kotalwar-krishna) [reply components/toast.css:2]: ok

**Pull request merged**